contracts/WildcardSteward_v0.sol

```diff
@@ -98,6 +98,13 @@ contract WildcardSteward_v0 is Initializable {
         benefactorFunds[msg.sender] = 0;
     }
 
+    function changeReceivingBenefactorAdmin(uint256 tokenId, address payable _newReceivingBenefactor) public onlyAdmin {
```

Comment:
It is not god to introduce a new function for this.
The old function with this modifier will work fine!
```
    modifier onlyReceivingBenefactorOrAdmin(uint256 tokenId) {
        require(msg.sender == benefactors[tokenId] || msg.sender == admin, "Not benefactor");
        _;
    }
```